### DISCUSSION TITLE
Implementation with Amber MD files (prmtop + nc)

### DISCUSSION THREAD

**Discussion by sribone** [scanberg/viamd#41] [Ideas]
Dear all, 
Have you thought on implementing the viamd for reading Amber MD files? Considering the high population of Amber users I think that would be great for this software popularity.
Thank you!!

**Comment by mathieulinares** [comment]
Thank you @sribone for your comment. We have been thinking about supporting Amber for a long time but did not get to it yet due to limited resources. We have been actively working on LAMMPS support that should be coming soon and Amber is next on our how to do list. We would be thrilled if members of the amber community could give us a hand to speedup the process.
Again, thanks for your feedback!

**Reply by sribone** [reply]
Thank you for your reply. I would try to put you in the way of the Amber developers.
Regards!!
